Adjust cluster users for addons (#7459)
kyma-project/kyma: docs/security/08-02-manage-static-users.md

By default, Kyma comes with the following ClusterRoles:

- **kyma-admin**: gives full admin access to the entire cluster

> - **kyma-namespace-admin**: gives full admin access except for the write access to AddonsConfigurations

Reviewer: ```suggestion
- **kyma-namespace-admin**: gives full admin access except for the write access to [AddonsConfigurations](https://kyma-project.io/docs/master/components/helm-broker#custom-resource-addons-configuration)
```

> - **kyma-namespace-admin**: gives an `kyma-admin` access without write permissions for AddonsConfigurations.

Reviewer: ```suggestion
- **kyma-namespace-admin**: gives full admin access except for the write access to AddonsConfigurations
```
